src/blacklist/5_0_0/sd/blacklistManagement/blacklistManagement_sd.tex

```diff
@@ -0,0 +1,338 @@
+\ArrowheadType{Service Description}
+\ArrowheadTypeShort{SD}
+\ArrowheadVersion{5.0.0} % Arrowhead version X.Y.Z, e..g. 4.4.1
+\ArrowheadDate{\today}
+\ArrowheadAuthor{Katinka Jakó} % Corresponding author e.g. Jerker Delsing
+\ArrowheadStatus{DRAFT} % e..g. RELEASE, RELEASE CONDIDATE, PROTOTYPE
+\ArrowheadContact{jako.katinka@aitia.ai} % Email of corresponding author
+\ArrowheadFooter{\href{www.arrowhead.eu}{www.arrowhead.eu}}
+\ArrowheadSetup
+%%
+
+%% Front Page
+\begin{center}
+  \vspace*{1cm}
+  \huge{\arrowtitle}
+
+  \vspace*{0.2cm}
+  \LARGE{\arrowtype}
+  \vspace*{1cm}
+
+  %\Large{Service ID: \textit{"\arrowid"}}
+  \vspace*{\fill}
+
+  % Front Page Image
+  %\includegraphics{figures/TODO}
+
+  \vspace*{1cm}
+  \vspace*{\fill}
+
+  % Front Page Abstract
+  \begin{abstract}
+    This document provides service description for the \textbf{blacklistManagement} service. 
+  \end{abstract}
+
+  \vspace*{1cm}
+
+%   \scriptsize
+%   \begin{tabularx}{\textwidth}{l X}
+%     \raisebox{-0.5\height}{\includegraphics[width=2cm]{figures/artemis_logo}} & {ARTEMIS Innovation Pilot Project: Arrowhead\newline
+%     THEME [SP1-JTI-ARTEMIS-2012-AIPP4 SP1-JTI-ARTEMIS-2012-AIPP6]\newline
+%     [Production and Energy System Automation Intelligent-Built environment and urban infrastructure for sustainable and friendly cities]}
+%   \end{tabularx}
+%   \vspace*{-0.2cm}
+ \end{center}
+
+\newpage
+%%
+
+%% Table of Contents
+\tableofcontents
+\newpage
+%%
+
+\section{Overview}
+\label{sec:overview}
+This document describes the \textbf{blacklistManagement} service, which enables systems (with operator role or proper permissions) to handle (query, create, remove) blacklist entries in bulk. An example of this interaction is when an operator bans a consumer because it is flooding the Local Cloud with thousands of requests. To enable other systems to use, to consume it, this service needs to be offered through the ServiceRegistry.
+
+The \textbf{blacklistManagement} service contains the following operations:
+
+\begin{itemize}
+    \item \textit{query} lists the entries that match the filtering requirements;
+    \item \textit{create} adds the specified entries to the blacklist;
+    \item \textit{remove} deactivates the entries that apply to the systems with the specified names.
+\end{itemize}
+
+The rest of this document is organized as follows.
+In Section \ref{sec:functions}, we describe the abstract message operations provided by the service.
+In Section \ref{sec:model}, we end the document by presenting the data types used by the mentioned operations.
+
+\subsection{How This Service Is Meant to Be Used}
+The service's purpose is to handle the blacklist entries centrally and in bulk. Thus, the \textbf{blacklistManagement} service makes it possible to manage who is (not) banned from the Local Cloud.
+
+Application systems should not use this service; only operators (via the Management Tool, for example) or dedicated Support systems.
```

Comment:
` ;` -> `,`

And I wouldn't reference here the Management Tool beacause it is not existing yet for v5, it might will have different name or there could be multiple tools like that. A more general phrase for example:
 "only human operators (indirectly via some tool, for example) or dedicated Support systems."